youtube_dl/extractor/zdf.py

```diff
@@ -39,10 +39,22 @@ def _extract_player(self, webpage, video_id, fatal=True):
 
 
 class ZDFIE(ZDFBaseIE):
-    _VALID_URL = r'https?://www\.zdf\.de/(?:[^/]+/)*(?P<id>[^/?]+)\.html'
+    IE_NAME = "ZDF-3sat"
+    _VALID_URL = r'https?://www\.(zdf|3sat)\.de/(?:[^/]+/)*(?P<id>[^/?]+)\.html'
```

Comment:
Please look at https://github.com/ytdl-org/youtube-dl/pull/21356#discussion_r299150796

Reply:
I'll try to find a suitable test case.

Reply:
https://www.3sat.de/wissen/nano/190822-sendung-100.html for 3SAT
https://www.zdf.de/kinder/wickie-und-die-starken-maenner/der-wegezoll-102.html for ZDF
both running with the patch above
download klappt mit dem Patch